examples/ring.rs

```diff
@@ -47,8 +51,9 @@ fn print_usage_and_exit() -> ! {
     ::std::process::exit(1);
 }
 
-#[actix_rt::main]
-async fn main() {
+fn main() -> std::io::Result<()> {
```

Comment:
Hm, this fix needs to remove `async`?

Reply:
Well, not really. The way that I got the example to run to completion instead of exiting prematurely was to do system.run() myself, because `node.send(Payload(0)).await.unwrap();` obviously wasn't waiting until all messages had been sent. So I removed `#[actix_rt::main]` to be able to call run().

Reply:
Ah, okay, indeed this fix is reasonable, I think.